toolkit/tools/pkgworker/pkgworker.go

```diff
@@ -14,7 +14,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/ccachemanager"
+	ccachemanagerpkg "github.com/microsoft/CBL-Mariner/toolkit/tools/internal/ccachemanager"
```

Comment:
is this change intentional?

Reply:
I've seen the same, `go fmt` prefers this. Was going to do the same next time I changed the file.

Reply:
Yes and no. The Go plug-in for VS Code adds these for cases where the "basename" of the package from the string doesn't match what's actually the package name where you go to the source. I can revert this line. I'm thinking about updating the package contents themselves to follow what we've been doing with other Go packages but that's a change for `main`.

Reply:
Created a separate PR here: #6958.